product-scenarios/5-route-messages-between-systems/5.3-load-balance-messages-among-systems/5.3.1-load-balance-messages-received-over-http/5.3.1.1-load-balance-with-load-balance-group/src/test/java/org/wso2/carbon/ei/scenario/test/LoadBalanceEndpointsWithLoadBalanceGroupTest.java

```diff
@@ -85,6 +86,11 @@ public void testLoadBalanceEndpointGroupWithNoPolicyNoAlgorithm() throws Excepti
         testLoadBalanceGroup("5_3_1_1_3_API_load_balance_group_define_nothing");
     }
 
+    @AfterClass(description = "Server Cleanup", alwaysRun = true)
```

Comment:
Do we need alwaysRun =true?

Reply:
This method should be run even if one or more methods invoked previously failed or was skipped. So we need alwaysRun=true